PortalTests/TableRendererSpec.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  TableRendererSpec.swift
+//  PortalTests
+//
+//  Created by Argentino Ducret on 9/5/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Portal
+
+class TableRendererSpec: QuickSpec {
+    
+    typealias ActionType = Action<MockRoute, String>
+    typealias CustomComponentRenderer = VoidCustomComponentRenderer<String, MockRoute>
+    typealias PortalTable = PortalTableView<String, MockRoute, CustomComponentRenderer>
+    
+    override func spec() {
+        
+        var layoutEngine: LayoutEngine!
+        
+        beforeEach {
+            layoutEngine = YogaLayoutEngine()
+        }
+        
+        describe(".apply(changeSet: TableChangeSet) -> Result") {
+            
+            var changeSet: TableChangeSet<ActionType>!
+            
+            beforeEach {
+                let itemPropertie: TableItemProperties<ActionType> = tableItem(height: 100, onTap: .sendMessage("OnTap!"), selectionStyle: .default) { _ in
+                    return TableItemRender(
+                        component: container(),
+                        typeIdentifier: "identifier"
+                    )
+                }
+                
+                let tableProperties: TableProperties<ActionType> = properties {
+                    $0.items = [itemPropertie, itemPropertie]
+                    $0.showsVerticalScrollIndicator = true
+                    $0.showsHorizontalScrollIndicator = false
+                }
+                
+                let style = tableStyleSheet { base, table in
+                    table.separatorColor = .red
+                }
+                
+                changeSet = TableChangeSet.fullChangeSet(
+                    properties: tableProperties,
+                    style: style,
+                    layout: layout()
+                )
+            }
+            
+            context("when the change set contains table property changes") {
+                
+                it("applies 'items' property changes") {
+                    let table = PortalTableView(layoutEngine: layoutEngine) {
```

Comment:
I see that you are repeating the initialization of table on every test, Why not initializing it on a beforeEach block?